src/content/platform-integration/platform-channels.md

```diff
@@ -8,14 +8,17 @@ description: Learn how to write custom platform-specific code in your app.
 
 This guide describes how to write custom platform-specific code.
 Some platform-specific functionality is available
-through existing packages;
-see [using packages][].
+through existing packages. For more information,
+visit [using packages][].
 
 [using packages]: /packages-and-plugins/using-packages
 
 :::note
-The information in this page is valid for most platforms,
-but platform-specific code for the web generally uses
+Starting in 3.29, Flutter on Android and iOS execute Dart code
```

Comment:
Likewise, for this section - changing the threads does not change how platform channels behaves. It will enable ffi interop in the future, but that is not done yet.